doc/source/whatsnew/v0.23.0.txt

```diff
@@ -240,7 +240,7 @@ The :func:`DataFrame.assign` now accepts dependent keyword arguments for python
   using ``.assign()`` to update an existing column. Previously, callables
   referring to other variables being updated would get the "old" values
 
-  Previous Behaviour:
+  Previous behavior:
```

Comment:
hmm, is the capitalization consistent across all of these entries? rather you leave this alone for now (and we can come back and clean it up later)